src/app/components/utility/modal/modal.component.ts

```diff
@@ -242,6 +277,57 @@ export class ModalComponent implements OnInit {
       this.isDisabled = e.target.value === this.user.last_name;
     } else if (e.target.name === 'update_affiliation') {
       this.isDisabled = e.target.value === this.user.affiliation;
+    } else if (e.target.name === 'update_google_scholar_url') {
+      if (this.validProtocol(e.target.value) && this.validURL(e.target.value)) {
```

Comment:
Can we please combine these (`validProtocol` and `validURL`) two functions into one for simplicity?